src/_data/lookingfor.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  en : {
+    heading: "Looking for another component?",
+    paragraph: "Check out our <a href=\"/en/components/\">component page</a>. If the component's not there, send us feedback on what you'd like us to work on next.",
```

Comment:
One small note: on the component overview page, it's supposed to say "Check out our coming soon page".